integration/run_test.go

```diff
@@ -291,8 +292,13 @@ func setUpISP(t *testing.T) (kubernetes.Interface, *v1.Namespace, func(t *testin
 	pubKey, privKey := testutil.CreateKeyPair(t, aaSecret)
 	// Create key secret in k8s cluster
 	createKeySecret(t, aaSecret, ns.Name, pubKey, privKey)
+
+	pgpKey, err := secrets.NewPgpKey(privKey, "", pubKey)
```

Comment:
I think there could be some problems with that approach. We may end up extending this function to create a PKIX key pair. Though we could deterministically compute an `id`, it may not match the `id` provided by the user, and thus it would go unused.